.eslintrc.json

```diff
@@ -1,19 +1,34 @@
 {
-	"extends": ["eslint:recommended", "prettier"],
+	"extends": [
+		"eslint:recommended",
+		"plugin:@typescript-eslint/eslint-recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:@typescript-eslint/recommended-requiring-type-checking",
+		"prettier",
+		"prettier/@typescript-eslint"
+	],
 	"env": {
 		"es6": true,
 		"node": true
 	},
+	"parser": "@typescript-eslint/parser",
 	"parserOptions": {
 		"ecmaVersion": 2018,
-		"sourceType": "module"
+		"sourceType": "module",
+		"project": ["./tsconfig.json", "./tsconfig.eslint.json"]
 	},
+	"plugins": ["@typescript-eslint"],
 	"rules": {
-		"no-unused-vars": [
+		"@typescript-eslint/explicit-function-return-type": "off",
+		"@typescript-eslint/no-var-requires": "off",
+		"@typescript-eslint/unbound-method": "off",
+		"@typescript-eslint/prefer-regexp-exec": "off",
+		"@typescript-eslint/no-unused-vars": [
 			"error",
 			{
 				"vars": "all",
-				"args": "after-used"
+				"args": "after-used",
+				"ignoreRestSiblings": false
```

Comment:
This doesn't need to be included, it appears `false` is the default value already.